packages/ListBox/src/components/Option/Option.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import useListBox from "../../useListBox";
+import { getA11yAttributesForOption } from "../../helpers/DOMAttributes";
+import { isOptionVisible, isOptionSelected, handleClickOption } from "../Options/helpers/options";
+import { OptionStyled } from "./Option.styles";
+
+const propTypes = {
+  /** String, number or JSX content */
+  children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]).isRequired,
+
+  /** Describe if the option is selected or not */
+  isSelected: PropTypes.bool,
+
+  /** Describe if the option is enable or not */
+  isDisabled: PropTypes.bool,
+
+  /** Describe if the option is hidden or not */
+  isHidden: PropTypes.bool,
+
+  /** When the children are not a String, label should need to be add so the filter can work  */
+  label: PropTypes.string,
+
+  /** Callback for the clicking event */
+  onClick: PropTypes.func,
+
+  /** Value of your option this can be any data structure  */
+  value: PropTypes.any, // eslint-disable-line
+};
+
+const defaultProps = {
+  isDisabled: false,
+  isHidden: false,
+  // internal prop, which shouldn't be documented
+  preventDefaultOnSelect: false, // eslint-disable-line
+  isSelected: false,
+  label: null,
+  onClick: null,
+  value: null,
+};
+
+export default function Option(props) {
+  const [state, dispatch] = useListBox();
+  const { activeOption, isDisabled: isDisabledState } = state;
+  const { index, groupId, label } = props; // eslint-disable-line
+
+  if (typeof state.options[index] === "undefined") {
+    return null;
+  }
+
+  if (!isOptionVisible(state, index)) {
+    return null;
+  }
+
+  /* eslint-disable react/prop-types */
+  const isSelected = isOptionSelected(state, index);
+  const isDisabled = isDisabledState || props.isDisabled;
+  const id = state.options[index].id;
+  return (
+    <React.Fragment>
+      <OptionStyled
+        {...getA11yAttributesForOption(state.options[index].isSelected)}
+        hasPreventDefaultOnSelect={props.preventDefaultOnSelect}
```

Comment:
✂️ snip? I think this is why the `RawItem` was implemented?

Reply:
the RawItem is an empty shell. This prop is still in play, was easier to just add the component to the consumer and keep the prop internally didn't have to modify the logic.